docs/src/creating-packages.md

```diff
@@ -13,6 +13,11 @@ This file is executed when the package is loaded.
 
 ### Generating files for a package
 
+!!! note
+    The [PkgTemplates](https://github.com/invenia/PkgTemplates.jl) package offers a very easy, repeatable, and 
+    customizable way to generate the files for a new package. We recommend that you use PkgTemplates for creating
+    new packages instead of using the bare-bones `pkg> generate` functionality described below.
```

Comment:
How about "minimal" instead of "bare-bones"? :grin:

Reply:
Done 😁